source/codegen/metadata_validation.py

```diff
@@ -317,6 +336,8 @@ def _validate_enum(enum_name: str, used_enums: Set[str], metadata: dict):
                     raise Exception(
                         f"generate-mappings is False, but values have non-int types: {value_types}"
                     )
+            if metadata["config"]["driver_name"] in _ALREADY_MUTATED:
```

Comment:
Note: This is to avoid the duplicate enum rule running on Attribute enums that will now be present in the MI driver metadata.

Reply:
Add this as a comment in the code.